buildenv/Build_Instructions_V8.md

```diff
@@ -348,12 +348,23 @@ tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2
 tar --one-top-level=/cygdrive/c/temp/freetype --strip-components=1 -xzf freetype-2.5.3.tar.gz
 ```
 
+Note:
+Please enure the freetype version supports win 32-bit application (e.g. freetype-2.4.7 which includes the "win32" folder)
```

Comment:
Please add something like "In order to build Windows 32-bit, please ensure ..."

Reply:
Already added.